tests/unit/AddStaffNotationDialog.spec.ts

```diff
@@ -326,24 +339,20 @@ describe('AddStaffNotationDialog', () => {
     sinon
       .stub(axios, 'post')
       .withArgs('businesses/BC0870669/filings')
-      .returns(
-        new Promise(resolve =>
-          resolve({
-            data: {}
-          })
-        )
-      )
+      .returns(new Promise(resolve => resolve({ data: { filing: {} } })))
```

Comment:
Now need to return data.filing due to response check in LegalServices.createFiling().